src/time_machine/__init__.py

```diff
@@ -117,6 +118,12 @@ def extract_timestamp_tzname(
         if dest.tzinfo is None:
             dest = dest.replace(tzinfo=dt.timezone.utc)
         timestamp = dest.timestamp()
+    elif isinstance(dest, dt.timedelta):
+        if escape_hatch.is_travelling():
+            raise TypeError(
+                "Timedelta destination is not supported when already time travelling."
```

Comment:
The name of the class is lowercase, so match that in error message.

```suggestion
                "timedelta destination is not supported when already time travelling."
```